src/main/java/duke/internal/Storage.java

```diff
@@ -0,0 +1,56 @@
+package duke.internal;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
```

Comment:
similarly, maybe auto ordering can be used here for consistent import statements